Data/Base/Shaders/Common/Lighting.h

```diff
@@ -470,9 +470,9 @@ void ApplyDecals(inout ezMaterialData matData, ezPerClusterData clusterData, uin
         }
         else
         {
-          decalWorldNormal = decalTangentNormal.x * worldToDecalMatrix._m00_m01_m02;
```

Comment:
Fix for decal size not working correctly when not using "map normal to geometry".